.gitignore

```diff
@@ -22,3 +22,4 @@ crt_portal/NO_ZIP.txt
 *.mo
 # this is where we store uploaded attachments when USE_LOCALSTACK=False
 attachments/
+README.md
```

Comment:
why README.md?

can you add output.csv instead, and remove output.csv from the committed files?

Reply:
yes! I just misunderstood the initial request